maintainers/maintainer-list.nix

```diff
@@ -7101,6 +7101,12 @@
     githubId = 15645854;
     name = "Brad Christensen";
   };
+  payas = {
```

Comment:
Could you please move the change of `maintainer-list.nix` to a separate commit?

Reply:
Ditto